package.json

```diff
@@ -8,6 +8,8 @@
   ],
   "scripts": {
     "start": "yarn --cwd examples/sandbox start",
+    "start-vue": "yarn --cwd examples/vue dev",
+    "start-docs": "cd ../docs.elastic.co/ && yarn init-docs && yarn dev",
```

Comment:
same suggestion as above: dont think it should live in the react-search-ui-views package. Likely better on the root package or question whether it needs to exist.

Reply:
It's already in the root package.json. 🙂 
I think it's convenient to be able to run all commands from the root of the repo rather than having to `cd` to specific directories.

Reply:
Oops! I think I read the next change as the location.